test/integration/features/step_definitions/testing-steps.js

```diff
@@ -0,0 +1,26 @@
+import {promises as fs} from 'fs';
+import {Then} from '@cucumber/cucumber';
+import {assert} from 'chai';
+
+Then('smoke tests are wired up', async function () {
+  const {devDependencies, scripts} = this.results;
+
+  ['cypress', 'start-server-and-test'].forEach(dependency => assert.include(devDependencies, dependency));
```

Comment:
you thinking of pulling this out to a test helper of some sort down the road? otherwise i think this is clearer to have them on separate lines.

Reply:
funny that you mention them being on separate lines... after adding `is-website-valueable` to the list, i had to break them onto separate lines in order to satisfy line-length. is it more in line with what you were thinking after that change?

Reply:
i was actually thinking making them separate assertions. when i read them i my eyes jumped down and assumed something more complex was going on so it took me a bit to read. definitely think they're valuable when there's a bunch, but when there's only a few i like to read them separately. just opinion, though

Reply:
gotcha. this wasnt my first choice. i initially tried doing something like `assert.include(devDependencies, [...]);`, but found that it doesnt work that way. there might be something other than `.include` that does if we dig a bit further, but i was in too much of a rush to break out of the zone at that point

Reply:
i was actually meaning the old-fashioned way 😄 :
```
assert.include(devDependencies, 'cypress')
assert.include(devDependencies, 'start-server-and-test')
```

Reply:
"unrolling" the loop essentially

Reply:
yeah, i followed what you were saying. i was just saying how i got there.

sounds like you are saying the list is more the problem than the loop though?